ci_config.json

```diff
@@ -190,6 +190,76 @@
       "libxkbcommon-dev"
     ]
   },
+  "fscl-xalgorithms-c": {
+    "fatal_warnings": true,
+    "skip_tests": true,
+    "build_on": {
+      "windows": false
+    }
```

Comment:
That's does do what you think it does. Fix your build. (This is to mark a project as not supported on Windows, and assumes you raise an error in your `meson.build`)

Reply:
Removed. This is a platform independent project(s)